components/CollectivePickerAsync.js

```diff
@@ -62,6 +62,9 @@ const Messages = defineMessages({
     id: 'Search',
     defaultMessage: 'Search',
   },
+  searchForUsers: {
+    defaultMessage: 'Search for Users by name or email',
```

Comment:
@Betree : Minor thing, but I was thinking if the word `Users` here should be capitalized. I don't think `users` is not a special term that we use and more broadly refers to all kind of users in the platform and in this case individuals. 🤔 

```suggestion
    defaultMessage: 'Search for users by name or email',
```

Reply:
This would go against consistency, we uppercase it with other parameter types (e.g. `types={[USER, COLLECTIVE]}` gives "Search for Users or Collectives"). See https://github.com/opencollective/opencollective-frontend/blob/4211d5db5e8ae713ad4cde124f2f290024713da1/lib/i18n/collective-type.js#L38